docs/tre-admins/start-stop.md

```diff
@@ -83,8 +83,33 @@ foreach ($Group in $ResourceGroups)
   {
     # Invoke the REST API
     Write-Output "Stopping $($Server.Name)"
-        $restUri='https://management.azure.com/subscriptions/'+$azContext.Subscription.Id+'/resourceGroups/'+$Group.ResourceGroupName+'/providers/Microsoft.DBForMySQL/servers/'+$Server.Name+'/stop?api-version=2020-01-01'
-        $response = Invoke-RestMethod -Uri $restUri -Method POST -Headers $authHeader
+    $restUri='https://management.azure.com/subscriptions/'+$azContext.Subscription.Id+'/resourceGroups/'+$Group.ResourceGroupName+'/providers/Microsoft.DBForMySQL/servers/'+$Server.Name+'/stop?api-version=2020-01-01'
+    $response = Invoke-RestMethod -Uri $restUri -Method POST -Headers $authHeader
+  }
+
+  $VMSS = Get-AzVMSS -ResourceGroupName $Group.ResourceGroupName
+  foreach ($item in $VMSS)
+  {
+    Write-Output "Stopping $($item.Name)"
+    Stop-AzVmss -ResourceGroupName $item.ResourceGroupName -VMScaleSetName $item.Name
+  }
+
+  $VM = Get-AzVM -ResourceGroupName $Group.ResourceGroupName
+  foreach ($item in $VM)
+  {
+    Write-Output "Stopping $($item.Name)"
+    Stop-AzVm -ResourceGroupName $item.ResourceGroupName -Name $item.Name
+  }
+
+  $WorkspaceResourceGroups = Get-AzResourceGroup -Name "$($Group.ResourceGroupName)-ws-*"
```

Comment:
workspace resource groups are also tagged with ```'project'='Azure Trusted Research Environment'``` so I think this second loop is redundant

Reply:
I changed the script a bit to not "catch" the workspace RGs in the outer loop since that might have unintended results down the line.